packages/next-app/src/app/metadata/pool/[poolId]/(components)/TransactionDialog.tsx

```diff
@@ -92,7 +94,9 @@ export function TransactionDialog({
     handleSubmit,
   } = useContext(PoolMetadataContext);
 
-  const { data: poolsData } = metadataGql.useMetadataPool({
+  const { data: poolsData } = metadataGql(
+    chain?.id.toString() || "1"
```

Comment:
why should we fallback to Ethereum here?

Reply:
the chain can be undefined, in our case, it won't be undefined since we just render this component for connected wallets. 
it's just to avoid typescript errors

Reply:
if you're sure it won't be undefined then the post-fix operator should be a better approach: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Reply:
You can also use `tiny-invariant`
```
import invariant from 'tiny-invariant';

invariant(chain.id, 'Chain empty :E');
```